package.json

```diff
@@ -4,6 +4,7 @@
   "description": "GitHub-like contribution calendar for the whole team. Supports GitHub and GitLab.",
   "main": "./lib/index.js",
   "scripts": {
+    "dev": "parcel ./dev/index.html --global TeamContributionCalendar -d ./lib/",
```

Comment:
I wouldn't pollute `lib` with parcel files even though it's completely ignored. The folder intends to contain the transpiled files, so it may be confusing to find additional ones.

How about `./dev/parcel`?

Reply:
Thanks for the suggestion, good idea.